chpl/chpl-resources/src/main/resources-dev/environment.properties

```diff
@@ -0,0 +1,133 @@
+questionableActivityNoDataEmailBody=<p>No questionable activity was found between %s and %s.</p>
+
+#chpl-service download jar properties
+dataSourceName=java:/comp/env/jdbc/openchpl
+
+#oversight email properties
+suspendedDaysAllowed=30
+capApprovalDaysAllowed=75
+capStartDaysAllowed=10
+ncOpenCapClosedDaysAllowed=45
+
+#oversight daily email properties
+oversightEmailDailySubject=Daily Surveillance Broken Rules Alert
+oversightEmailDailyHtmlMessage=<h3>ONC Daily Surveillance Report</h3>
+oversightEmailDailyFileName=surveillance-oversight-daily-report-
+oversightEmailDailyNoContent=<p>No surveillance oversight rules were newly broken in the last day</p>
+oversightEmailAcbDailyHtmlMessage=<h3>ONC-ACB Daily Surveillance Report</h3>
+
+#oversight weekly email properties
+oversightEmailWeeklySubject=Weekly Surveillance Broken Rules Alert
+oversightEmailWeeklyHtmlMessage=<h3>ONC Weekly Surveillance Report</h3>
+oversightEmailWeeklyFileName=surveillance-oversight-weekly-report-
+oversightEmailWeeklyNoContent=<p>No surveillance oversight rules are broken</p>
+oversightEmailAcbWeeklyHtmlMessage=<h3>ONC-ACB Weekly Surveillance Report</h3>
+
+#inheritance report weekly email properties
+inheritanceReportEmailWeeklySubject=Weekly ICS Validation Report
+inheritanceReportEmailWeeklyHtmlMessage=<h3>ONC Weekly ICS Validation Report</h3>
+inheritanceReportEmailWeeklyFileName=inheritance-validation-weekly-report-
+inheritanceReportEmailWeeklyNoContent=<p>No listings were found that do not meet ICS requirements.</p>
+inheritanceReportEmailAcbWeeklyHtmlMessage=<h3>ONC-ACB Weekly ICS Validation Report</h3>
+
+#listing upload error email properties
+uploadErrorEmailRecipients=382e04a9.ainq.com@amer.teams.ms
+uploadErrorEmailSubject=CHPL Listing Upload Error
+
+#schema download file properties
+schemaXmlName=listingSchema.xsd
+schemaCsv2014Name=2014 Listing CSV Data Dictionary.csv
+schemaCsv2015Name=2015 Listing CSV Data Dictionary.csv
+schemaSurveillanceName=Surveillance and Nonconformity CSV Data Dictionary.csv
+
+#chpl-auth properties
+authJndiName=java:comp/env/jdbc/openchpl
+authPersistenceUnitName=openchpl
+authAclCacheName=aclCache
+authMaximumLoginAttempts=5
+keyLocation=/opt/tomcat8/tomcat-latest/conf/JSONRsaJoseJWebKey.txt
+jwtAllowedClockSkew=30
+jwtIssuer=ONCCHPL
+jwtAudience=ONCCHPL
+jwtExpirationTimeMinutesInTheFuture=30
+jwtNotBeforeMinutesInThePast=4
+resetLinkExpirationTimeInHours=1
+
+####### Email Builder Settings #######
+#Do we want to 'check' emails before sending them?
+#emailBuilder.config.productionEnvironment=false
+#Let emails being sent to this list of domains go through
+#emailBuilder.config.whitelistedDomains=ainq.com,onc.gov
+#Emails should be sent to this address
+#emailBuilder.config.forwardAddress=example@ainq.com
+#######################################
+emailBuilder.config.productionEnvironment=false
+emailBuilder.config.whitelistedDomains=ainq.com
+emailBuilder.config.forwardAddress=78e7678e.ainq.com@amer.teams.ms
```

Comment:
Should make this a parameter too. Definitely prefer not to have random web scrapers emails our channel...